### PR TITLE
send_null pattern

### PR DESCRIPTION
**Pull request by silverCase**

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#187](https://github.com/yegor256/aibolit/issues/187) is assigned to @lyriccoder[/z](https://www.0crat.com/u/lyriccoder), here is [why](https://www.0crat.com/footprint/CT5UB7XAP/fcffb352-4e9b-4730-b3ea-4dbfb68a3c6c); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @acheshkov[/z](https://www.0crat.com/u/acheshkov) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

File: aibolit/patterns/send_null/send_null.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import javalang
+from aibolit.utils.ast import AST
+
+
+class SendNull:
+
+    def __init__(self):
+        pass
+
+    def value(self, filename: str):
```
Comment (acheshkov) [inline aibolit/patterns/send_null/send_null.py:10]: I recommend you to add typings to your code. Use typings for method signatures and for declared variables. See more examples in the other patterns.

```
from typing import List
...
def value(self, filename: str) -> List[int]: 
  list: List[int[ = []
...
```

**Comment by acheshkov** [comment]
@silverCase add pattern description to PATTERNS.md

File: aibolit/patterns/send_null/send_null.py (hunk 1 above)
Comment (silverCase) [reply aibolit/patterns/send_null/send_null.py:10]: Will do

**Comment by 0crat** [comment]
Job was finished in 21 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
Job audit: performer @lyriccoder[/z](https://www.0crat.com/u/lyriccoder)[/z](https://www.0crat.com/u/lyriccoder) didn't make CR comments

**Comment by 0crat** [comment]
Pull request [#187](https://github.com/yegor256/aibolit/issues/187) was not merged, no payment for `ARC`, see [§28](http://www.zerocracy.com/policy.html#28)

**Pull request closed without merging**